modules/farm_fd2_school/src/entrypoints/FD3_Application/App.vue

```diff
@@ -170,6 +167,7 @@ export default {
       this.comment = '';
     },
     async submitForm() {
+      console.log('App: submitForm() called');
```

Comment:
`console.log` statements used for debugging should be removed before a PR is made.